corehq/apps/notifications/models.py

```diff
@@ -34,7 +36,8 @@ class Meta(object):
         ordering = ["-activated"]
 
     @classmethod
-    def get_by_user(cls, django_user, couch_user, limit=10):
+    def get_by_user(cls, django_user, couch_user, limit=10, plan='Community',
+                    has_cs_groups=False, is_USH_or_Solutions=False):
```

Comment:
Rather than include Dimagi specific logic like this (is_USH_or_Solutions), it'd be better to keep it agnostic. So I'd ask what is it that changes if that value is true or false as far as notifications go? It looks like the answer is feature discoverability notifications are not included if that value is true, so how about renaming to should_exclude_features or something similar?